src/components/Login/ExistedAccountPwdSetting/index.tsx

```diff
@@ -0,0 +1,135 @@
+import ModalOperator from "@/components/Login/ModalOperater";
+import {
+  Button,
+  Divider,
+  Flex,
+  Icon,
+  ModalBody,
+  ModalHeader,
+  Text,
+  useToast,
+} from "@chakra-ui/react";
+import MainLogoSvg from "@/assets/svg/CourtCanva-main-LOGO.svg";
+import PwdInputGroup from "@/components/Login/PwdInputGroup";
+import React, { useState } from "react";
+import { updateUser } from "@/components/Login/helpers/userRequests";
+
+type Props = {
+  onClose: any;
```

Comment:
It can be written like this: _onClose: () => void_
Avoid using _any_ as value type

Reply:
Thanks, Shawn! I've resolved this problem!